esmvaltool/recipes/recipe_hyintex.yml

```diff
@@ -37,6 +38,12 @@ diagnostics:
     scripts:
       main:
         script: hyint/hyint.R
+        # General
+        run_diagnostic: true
```

Comment:
Most of these options should be set in [config-user.yml](https://github.com/ESMValGroup/ESMValTool/blob/version2_development/esmvaltool/config-user.yml) instead of in the recipe.